src/components/DisplayTodo.js

```diff
@@ -1,16 +1,30 @@
 // eslint-disable-next-line
 import React from 'react';
+// eslint-disable-next-line
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+// eslint-disable-next-line
+import { icon } from '@fortawesome/fontawesome-svg-core/import.macro'; // <-- import styles to be used
 
-const DisplayTodo = () => {
+const DisplayTodo = (props) => {
+  const { onHandleCheckBoxChange, onDeleteHandler } = props;
   const todos = JSON.parse(localStorage.getItem('todos'));
 
-  const todoList = todos.map((todo) => (
-        <div className='content-container' key={todo.id}>
-            <input type='checkbox' />
-            <p className='content-paragraph'>{todo.content}</p>
+  const onhandleDelete = (id) => {
+    onDeleteHandler(id);
+  };
 
+  let todoList = null;
+
+  if (todos !== null) {
+    todoList = todos.map((todo) => (
+        <div className='content-container' key={todo.id}>
+          <input id={todo.id} onChange={() => { onHandleCheckBoxChange(todo.id); }} type='checkbox' />
+          <p className='content-paragraph'>{todo.content}</p>
+          <FontAwesomeIcon onClick={() => { onhandleDelete(todo.id); }} className='trash-icons' icon={icon({ name: 'trash', family: 'sharp', style: 'solid' })} />
```

Comment:
The icon you added here is a Pro icon, which isn't part of the free kit provided by FontAwesome. In order for the project to run on my machine, I removed the family property from the object you passed to `icon` like this: 
`icon={icon({ name: 'trash', style: 'solid' })}`